storage/PreloaderThread.cpp

```diff
@@ -49,37 +49,46 @@ void PreloaderThread::run() {
   std::size_t blocks_loaded = 0;
 
   for (const CatalogRelation &relation : database_) {
-    if (relation.hasPartitionScheme()) {
+    if (relation.hasPartitionScheme() && relation.hasNUMAPlacementScheme()) {
```

Comment:
Thanks for pointing out @jianqiao. I can remove the check in another PR. However I wonder, why the function hasNUMAPlacementScheme() should be unavailable when NUMA support is not present. It should just return false then. cc @adalbertgerald